karton/misp_pusher/misp_pusher.py

```diff
@@ -51,10 +62,39 @@ def process(self, task: Task) -> None:
             # Nothing actionable found - skip the config
             return
 
+        misp = ExpandedPyMISP(
+            http_url(self.config.get("misp", "url")),
+            self.config.get("misp", "key"),
+            not self.config.getboolean("misp", "insecure", False),
+        )
+
         # Upload structured data to MISP
         event = MISPEvent()
         event.uuid = str(uuid5(self.CONFIG_NAMESPACE, dhash))
+
         event.add_tag(f"mwdb:family:{family}")
+
+        if self.cluster_mapping:
+            if family not in self.cluster_mapping:
```

Comment:
Do we want to fail hard in this case? Maybe we should just continue processing the event here (and maybe log a warning)